extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1289,3 +1290,100 @@ export const SQLDB_COL_COPY_DURATION = localize('sql.migration.sqldb.column.copy
+export const TDE_WIZARD_MSG_MANUAL = localize('sql.migration.tde.msg.manual', "You have chosen to manually migrate certificates.");
+export const TDE_WIZARD_MSG_TDE = localize('sql.migration.tde.msg.tde', "You have given access to Azure Data Studio to migrate the encryption certificates and database.");
+export const TDE_WIZARD_MSG_EMPTY = localize('sql.migration.tde.msg.empty', "No encrypted database selected.");
+
+export function TDE_MIGRATION_ERROR(message: string): string {
+	return localize('sql.migration.starting.migration.error', "An error occurred while starting the certificate migration: '{0}'", message);
+}
+
+export function TDE_MIGRATION_ERROR_DB(name: string, message: string): string {
+	return localize('sql.migration.starting.migration.dberror', "Error migrating certificate for database {0}. {1}", name, message);
+}
+
+export function TDE_MSG_DATABASES_SELECTED(selected: number, message: string): string {
+	return localize('sql.migration.tde.msg.databases.selected', "{0} Transparent Data Encryption enabled databases selected for migration. {1}", selected, message);
+}
+
+export function TDE_WIZARD_DATABASES_SELECTED(encryptedCount: number, totalCount: number): string {
+	return localize('sql.migration.tde.wizard.databases.selected', "{0} out of {1} selected database(s) is using transparent data encryption.", encryptedCount, totalCount);
+}
+
+
+export const TDE_WIZARD_MIGRATION_OPTION_MANUAL_WARNING = localize('sql.migration.tde.wizard.optionmanual.warning', "You must migrate the certificates before proceeding with the migration otherwise the migration will fail. {0}.");
+
+export const TDE_WIZARD_ADS_CERTS_INFO = localize('sql.migration.network.share.header.text', "Please enter a location where the SQL Server will export the certificates. Also verify that SQL Server service has write access to this path and the current user should have administrator privileges on the computer where this network path is.");
+
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_LABEL = localize('sql.migration.tde.wizard.network.share.label', "Network path for certificate");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_PLACEHOLDER = localize('sql.migration.tde.wizard.network.share.placeholder', "Enter network path");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_INFO = localize('sql.migration.tde.wizard.network.share.info', "Network path where certificate will be placed.");
+
+export const TDE_MIGRATE_BUTTON = localize('sql.migration.tde.button.migrate', "Migrate certificates");
+
+
+export const STATE_CANCELED = localize('sql.migration.state.canceled', "Canceled");
+export const STATE_PENDING = localize('sql.migration.state.pending', "Pending");
+export const STATE_RUNNING = localize('sql.migration.state.running', "Running");
+export const STATE_SUCCEEDED = localize('sql.migration.state.succeeded', "Succeeded");
+export const STATE_FAILED = localize('sql.migration.state.failed', "Failed");
```

Comment:
can you use StatusLookup for this or replicate the pattern to represent a tde_migration_state

Reply:
This is being used following the StatusLookup pattern 
![image](https://user-images.githubusercontent.com/109680247/212173307-96a44a00-d096-4f9d-bc4a-5e0677f323bd.png)

I removed the TDE prefix from the names to make them generic based on a previous comment from Raymond to reuse some localized strings.